package.json

```diff
@@ -50,7 +50,7 @@
     "url-loader": "^4.1.1",
     "webpack": "^5.37.1",
     "webpack-cli": "^4.7.2",
-    "webpack-dev-server": "^3.11.2"
+    "webpack-dev-server": "^4.0.0-beta.3"
```

Comment:
Why this change? If we upgrade to v4, I'd prefer going straight to 4.0.0-rc.0 since 4.0.0-beta.3 has some breaking changes when compared with the newer rc.0

Reply:
We used vers 4.0.0-beta.3 as part of debugging to get my docker logging / watcher to work.  Final fix was closing the number of terminals I had open and restarting the computer.  Should I go back to 3.11.2 or 4.0.0-rc.0?

Reply:
Let's go to 4.0.0-rc.0